src/views/EditView.js

```diff
@@ -17,8 +17,8 @@
       this.on("viewDeactivate",this.viewDeactivate);
       this.model.bind("all", this.addAll, this);
       this.confirmBackNav = {
-        title: "Confirm navigation",
-        subtitle: "Discard any changes?",
+        title: "You have an unsaved item",
+        subtitle: "Go back and discard this new item?",
```

Comment:
This is for editing as well as adding... "new item" might not be what you are after...